tests/controller_tests/test_filters.py

```diff
@@ -89,6 +89,51 @@ def test_get_funds(seeded_test_client):
     assert all(isinstance(fund["id"], str) for fund in response_json)
 
 
+class MockProgramme:
+    def __init__(self, fund_type_id):
+        self.fund_type_id = fund_type_id
+
+
+class FundTypeIdEnum:
+    TOWN_DEAL = "TD"
+    HIGH_STREET_FUND = "HS"
+
+
+def test_get_funds_alphabetically(seeded_test_client):
+    """
+    Test the function that retrieves funds in alphabetical order.
+
+    :Args:
```

Comment:
as a general rule, you don't really need to add the args and raises to a test if you don't want since the function is not used by anything (not asking to do anything, just a general observation)